### DISCUSSION TITLE
S3 Configuration Screen Questions

### DISCUSSION THREAD

**Discussion by hellosimplerick** [iterate-ch/cyberduck#15605] [Q&A]
is there any S3 documentation that specifically shows how to set up a new connection with an explanation of each field so I can better understand what is being requested and then go find it.  This is for bother Mountain and Cyber. I'm super new.  I have an S3 bucket setup.  Thanks in advance for any help.

<img width="444" alt="Screenshot 2024-02-01 at 10 58 16" src="https://github.com/iterate-ch/cyberduck/assets/91222539/98721f55-6abd-4b80-9bb5-7ffc498c816b">

**Comment by dkocher** [comment]
Refer to our [documentation](https://docs.cyberduck.io/protocols/s3/).
